mcp/src/main/java/io/modelcontextprotocol/client/McpAsyncClient.java

```diff
@@ -780,16 +780,15 @@ private NotificationHandler asyncLoggingNotificationHandler(
 	 * @return A Mono that completes when the logging level is set.
 	 * @see McpSchema.LoggingLevel
 	 */
-	public Mono<Void> setLoggingLevel(LoggingLevel loggingLevel) {
+	public Mono<Object> setLoggingLevel(LoggingLevel loggingLevel) {
```

Comment:
Why `Object` and not an actual meaningful type?

Reply:
because there is not response. It is the same request as the ping.

Reply:
Ok, it's an empty response. Got it.